### PR TITLE
add menu's front end

### PR DESCRIPTION
**Pull request by IanWen103026**
add images
add custom.css
edit application.html.erb
edit menu/index.html.erb

### CODE REVIEW

**Comment by ckeisc43rd-cjtsai** [comment]
the container should stay in application.html.erb as stated earlier, explicit <script> to external .js file

File: app/views/menus/source/.DS_Store
Comment (AaronWu-train) [inline app/views/menus/source/.DS_Store:1]: Remove this file

File: app/views/menus/index.html.erb [hunk 1]
```diff
@@ -1 +1,94 @@
...
+</script>
+<script>
+resizebigcontainer();
+
+    $(window).on('load resize', function() {
+        resizebigcontainer();
+    });
+
+    function resizebigcontainer() {
+        var bigcontainer = $('.bigcontainer');
```
Comment (AaronWu-train) [inline app/views/menus/index.html.erb]: Change to call by id

File: app/views/menus/index.html.erb [hunk 2]
```diff
@@ -1 +1,94 @@
...
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="stylesheet" href="styles.css">
+<link href="jquery-sakura.css" rel="stylesheet" type="text/css">
+
+</head>
+<body>
+
+    <div class="bigcontainer">
```
Comment (AaronWu-train) [inline app/views/menus/index.html.erb]: use id
rename class name to a more precise name

File: app/assets/stylesheets/custom.css [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+
+    height: 8vw;
+    width: 8vw;
+    border-radius: 50%; /* Make it a circle */
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+#cjtsai {
```
Comment (AaronWu-train) [inline app/assets/stylesheets/custom.css]: WTF is this

**Review by AaronWu-train — CHANGES_REQUESTED** [review]
Please Change the gitignore too

**Review by AaronWu-train — APPROVED** [review]
LGTM

**Pull request merged**